README.md

```diff
@@ -131,6 +131,18 @@ Uncovered API definitions found:
 /v2/pet/:petId/uploadImage | POST       | 200         
 ```
 
+## Custom Content-Type support
```

Comment:
Will remove this part, since plan is to support all content-type by default. However, being able to override schema building options might still be useful.